data/implementations/discovery.json

```
@@ -39,6 +44,11 @@
           "name": "Go",
           "status": "Done",
           "url": "https://github.com/libp2p/go-libp2p-kad-dht"
+        },
+        {
+          "name": "Rust",
+          "status": "Unstable",
```

I think my comment got lost (sorry in case I am missing something): We use [random walks](https://github.com/paritytech/substrate/blob/master/client/network/src/discovery.rs#L29) in substrate heavily. Thus I would suggest changing this to `Usable`.
```suggestion
          "status": "Usable",
```